hummingbot/strategy/order_tracker.pxd

```diff
@@ -25,11 +25,11 @@ cdef class OrderTracker(TimeIterator):
     cdef LimitOrder c_get_limit_order(self, object market_pair, str order_id)
     cdef object c_get_market_order(self, object market_pair, str order_id)
     cdef LimitOrder c_get_shadow_limit_order(self, str order_id)
-    cdef c_start_tracking_limit_order(self, object market_pair, str order_id, bint is_buy, object price,
```

Comment:
I believe the reason two different tracked orders are created (`_tracked_limit_orders` and `_tracked_market_orders`) is to support 2 different data structures (market order type has no price). Since we no longer have market order type, there should be just one list `tracked_orders`.

Reply:
This would also remain.